AUTORECONF
PATCHLEVEL=0
Index: Makefile.am
===================================================================
--- Makefile.am	(revision 2640)
+++ Makefile.am	(working copy)
@@ -12,14 +12,14 @@
 
 
 install-data-hook:
-	$(mkinstalldirs) $(docdir)
-	cp README NEWS COPYING INSTALL AUTHORS ChangeLog $(docdir)
-	rm -rf $(docdir)/CVS
-	rm -rf $(docdir)/*/CVS
-	rm -rf $(docdir)/*/*/CVS
+	$(mkinstalldirs) $(DESTDIR)$(docdir)
+	cp README NEWS COPYING INSTALL AUTHORS ChangeLog $(DESTDIR)$(docdir)
+	rm -rf $(DESTDIR)$(docdir)/CVS
+	rm -rf $(DESTDIR)$(docdir)/*/CVS
+	rm -rf $(DESTDIR)$(docdir)/*/*/CVS
 
 uninstall-hook:
-	rm -rf $(docdir)
+	rm -rf $(DESTDIR)$(docdir)
 
 
 dist-hook:
@@ -38,10 +38,10 @@
 
 doc-install:
 	$(MAKE) $(AM_MAKEFLAGS) doc doc-all
-	(cd doc ; cp -r * $(docdir) )
-	rm -rf $(docdir)/CVS
-	rm -rf $(docdir)/*/CVS
-	rm -rf $(docdir)/*/*/CVS
+	(cd doc ; cp -r * $(DESTDIR)$(docdir) )
+	rm -rf $(DESTDIR)$(docdir)/CVS
+	rm -rf $(DESTDIR)$(docdir)/*/CVS
+	rm -rf $(DESTDIR)$(docdir)/*/*/CVS
 
 doc-all: doc-pdf doc-html-archive
 
